2019-08-13 14:54:33,615 ERROR: Exception on /reset_password_request [POST] [in d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\MyProjects\flask_tutorial_projects\microblog\app\routes.py", line 164, in reset_password_request
    return render_template(url_for('login'))
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\jinja2\loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /login
2019-08-13 14:55:12,818 ERROR: Exception on /reset_password_request [POST] [in d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\MyProjects\flask_tutorial_projects\microblog\app\routes.py", line 164, in reset_password_request
    return render_template(url_for('login'))
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\jinja2\loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /login
2019-08-13 14:57:43,174 ERROR: Exception on /reset_password_request [POST] [in d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\MyProjects\flask_tutorial_projects\microblog\app\routes.py", line 164, in reset_password_request
    return render_template(url_for('login'))
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\jinja2\loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "d:\myprojects\flask_tutorial_projects\microblog\venv\lib\site-packages\flask\templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /login
2019-08-13 15:01:50,612 INFO: Microblog startup [in D:\MyProjects\flask_tutorial_projects\microblog\app\__init__.py:48]
